src/components/SingleList.jsx

```diff
@@ -1,19 +1,102 @@
-import { useNavigate } from 'react-router-dom';
 import './SingleList.css';
+import { useNavigate } from 'react-router-dom';
+import { useState } from 'react';
+import PushPinIcon from '@mui/icons-material/PushPin';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { MaterialListButton, MaterialButton } from './material-buttons';
+import { useAuth, deleteList } from '../api';
+import { ConfirmDialog } from './ConfirmDialog';
+
+export const deletionResponse = {
+	hard: `List deleted permanently.`,
+	soft: `List removed from user view.`,
+};
+
+export function SingleList({
+	item,
+	setListPath,
+	setImportantList,
+	isImportant,
+}) {
+	const [isHovered, setIsHovered] = useState(false);
+	const [open, isOpen] = useState(false);
 
-export function SingleList({ name, path, setListPath }) {
 	const navigate = useNavigate();
+	const { user } = useAuth();
```

Comment:
Consider moving these hooks above line 21 + 22.

Hooks should always be first in the component level above stuff like `useState`. Check out rules of hooks npm packages for some more details (no need to install on this project if you don't want to or have the time) https://www.npmjs.com/package/eslint-plugin-react-hooks

Reply:
Will do!